src/acquisition/covidcast/csv_to_database.py

```diff
@@ -28,16 +28,21 @@ def get_argument_parser():
   parser.add_argument(
     '--log_file',
     help="filename for log output (defaults to stdout)")
+  parser.add_argument(
+    '--indicator_name',
+    nargs='?',
+    default='*',
+    help='Name of one indicator directory to run acquisition on')
   return parser
 
 
-def collect_files(data_dir: str, specific_issue_date: bool):
+def collect_files(data_dir: str, specific_issue_date: bool, indicator_name = "*"):
   """Fetch path and data profile details for each file to upload."""
   logger= get_structured_logger('collect_files')
   if specific_issue_date:
     results = list(CsvImporter.find_issue_specific_csv_files(data_dir))
   else:
-    results = list(CsvImporter.find_csv_files(os.path.join(data_dir, 'receiving')))
+    results = list(CsvImporter.find_csv_files(os.path.join(data_dir, 'receiving'), indicator_dir = indicator_name))
```

Comment:
```suggestion
    results = list(CsvImporter.find_csv_files(os.path.join(data_dir, 'receiving'), indicator_dir=indicator_name))
```
nit: when specifying keyword arguments, the convention is to omit spaces around the equals